tensorflow_datasets/image_classification/mnist.py

```diff
@@ -264,7 +264,7 @@ class EMNIST(MNIST):
       ),
       EMNISTConfig(
           name="letters",
-          class_number=37,
+          class_number=26,
```

Comment:
@sourabhsingh282 This PR is duplicate of #2591. It would be best to close this and keep that one.